Description: Workaround for a format string that generate a compilation warning
 transformed to an error by Werror option of gcc.
Author: fabien boucher <fabien.dot.boucher@gmail.com>

Index: json-c-0.10/tests/test_printbuf.c
===================================================================
--- json-c-0.10.orig/tests/test_printbuf.c	2012-04-29 17:55:43.000000000 +0000
+++ json-c-0.10/tests/test_printbuf.c	2012-08-16 21:03:09.000000000 +0000
@@ -124,7 +124,7 @@
 	memset(data, 'X', before_resize + 1 + 1);
 	data[before_resize + 1] = '\0';
 	sprintbuf(pb, "%s", data);
-	printf("sprintbuf to just after resize(%d+1): %d, [%s], strlen(buf)=%d\n", before_resize, printbuf_length(pb), pb->buf, strlen(pb->buf));
+	printf("sprintbuf to just after resize(%d+1): %d, [%s], strlen(buf)=%lu\n", before_resize, printbuf_length(pb), pb->buf, (unsigned long)strlen(pb->buf));
 
 	printbuf_reset(pb);
 	sprintbuf(pb, "plain");
